Update ThreatQ README with tested versions (#25696)
demisto/content: Packs/ThreatQ/Integrations/ThreatQ_v2/README.md

<!-- HTML_DOC -->
<p>A threat intelligence platform that collects and interprets intelligence data from open sources and manages indicator scoring, types, and attributes.</p>

> <p>This integration has been tested and works with ThreatQ API in versions 4 and 5</p>

Reviewer: ```suggestion
<p>This integration was integrated and tested with API versions 4 and 5 of ThreatQ.</p>
```
